apps/cli/tests/test_pending_visibility.py

```
@@ -0,0 +1,320 @@
+) -> None:
+    """Rich reports a closed stream as ValueError and a broken pipe as SystemExit."""
+    _buffer(1)
+
+    for boom in (
+        ValueError("I/O operation on closed file"),
+        SystemExit(1),
+        BrokenPipeError(),
+    ):
+
+        def explode(*_args: object, **_kwargs: object) -> None:
+            raise boom
```

Ruff (0.16.1)

[warning] Function definition does not bind loop variable `boom`

(B023)